lib/tasks/i18n_cov.rake

```diff
@@ -1,16 +1,18 @@
-require "i18n/coverage/printers/file_printer"
+unless Rails.env.production?
```

Comment:
You can do `return unless Rails.env.production?` to save nesting the whole file.

I'd suggest doing `unless defined?(I18n::Coverage)` rather than the more circumstantial environment (assuming that at this point bundler has already required files)

Reply:
Sure, I've updated and run a test deploy in Integration.